chunk/chunk_store.go

```diff
@@ -206,6 +208,10 @@ type AWSStore struct {
 	tableName  string
 	bucketName string
 
+	// Around which time to start bucketing indexes by day instead of by hour.
+	// Only the day matters, not the time within the day.
+	dailyBucketsFrom model.Time
```

Comment:
We can make it an int, but this just pushes the problem into main.go.  I think I prefer it like this.